src/utils/locale.ts

```diff
@@ -1,4 +1,5 @@
 import { sanitizeDecimalString, sanitizeExponentialNumberString, isValidNumber, BigDecimal } from "./number";
```

Comment:
Wondering if we should rename this file to `lang.ts`?

Reply:
Yeah good call

Reply:
I thought about that, and I believe we can keep using `locale` in our doc and internally for consistency across product terminology. The way I'm thinking about this is we're deriving locale from the language, per the spec referenced in the description.

Reply:
I like that.  `locale` is a more specific term too, because it includes both language and the region where the language is spoken.